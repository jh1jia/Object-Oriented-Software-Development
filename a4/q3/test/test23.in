test22.txt dropfirst 0 allcaps
