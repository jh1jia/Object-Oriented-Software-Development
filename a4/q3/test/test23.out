DDDDDDDD
ABCDEFGH
D023AAC3
45%DC
C
C
CC
CC
1DD20#DCE00
00
